samples/C4CoreDiagrams.md

```diff
@@ -44,7 +44,7 @@ Source: [C4_Dynamic Diagram Sample - bigbankplc.puml](C4_Dynamic%20Diagram%20Sam
 
 Source: [C4_Deployment Diagram Sample - bigbankplc.puml](C4_Deployment%20Diagram%20Sample%20-%20bigbankplc.puml)
 
-![Deployment Diagram for Internet Banking System - Live](https://www.plantuml.com/plantuml/png/rLPBRziu4BxhLypwaWSIkp62HT5J73lR96l7HkKqBR20CI9fcLkA54ZAhhVeV-zGiYChSO2cFQqFWZZcDuzlNhfm7gqlCXNz8JLJ1HUmy3vtxpeTYwjsAlsYI0edB3FQ2-tRp6GTv5QYNgeYBphzyzE9GktlHyEEGY-S9z5i879brba0v47v0HIQa440VY5WRfGoAwbJK58BGCsXyRUqZjuCa6LGXg5IQzBnwE8U2z71iFTvV3-TzU9P_FVLj7yTRGGFDzFhsV3g_TNTuF0eYhpqIa0DXu74r696FbcueI2j5XukKNy9xiLhvqK6fp2KIn55DMnsPxWup1KxWTQbJ4i0qB7rv0mSFG8JGN7hw5i4z7kgXMi29J9D27EAkJm-Flx7VtqRLDqdXVO5dFtPVdCEmscyLR5F3UR85e90lV85fYPZw5jF1V2s_NLNJVZrARj8cR2aH0R0v0PwUQuatPT61nstkCGoKN5k0pEZZ_qWdbYpb5ouvzJDAUK1Z4hwDImbeU4s7jzrhgVJIGnaejqwUlJYUrG_dmQseOOdRyvsY0h7rr9aB38LA1gNBvJNYPKPsdMhbfrrsOlq3993dj3L0IKrGIS2x8DGMCK9Mp74H7U6uT_O6qliX1O2ArBfl2rlWzHKS5dvVW8BT0l10GkgVkqth0EpWeSZAdS2o9XmZheXfOFmxFNaTPlaTN-DlEukUx6WkS1VoL_tVqPWialSIYHLUvx_MdjsVxC_oJu17aJoa_qv48ecbtNbNALXJzp0PfQNjuFuMUiwUb0cSqnfDjTczZ7s_VbGp4nmiLyuRp9X3noGG98wBfQIYIhUdCutunXevlOqjqRoRSXrg1MCheygFA42-1DImqhT63v8UoIJdYh05OcZICnyVsJDeKGRXctSTEM48eoqJNz3q7k2f7MLDwCCaY9C-U2nf8NfpDp3WzJSh5nmg8ewGpQENze4YQMxmcuoV5cTWiBtneICz1UMh9t0oFmdbKBuIrenDurogVbpEOR-gN8v2RdSKp8nipAlIg6dqqAXRHAhrEznIVKAcwYsvLCM3JNzOuAhdJF2BsGmR9wmxwbWJV3aNsTqjJ487K1bLAz2b2rJaL5tWobZOdNwTXpTu4eZvRZ_890xc1Ft9L4hIwMsDNSxkEmt0TsVHmJjMq2L6hfLVZ7GLVfOkH2QqzVD3m00 "Deployment Diagram for Internet Banking System - Live")
+![Deployment Diagram for Internet Banking System - Live](http://www.plantuml.com/plantuml/png/rLNVJoD737xFN-7cXHy2SaIeElNfGa8FA87eDfHMgXHvPvpDz6PdLZEpWRIw_xssDX0MKkcuUxeyH3jUVxQ_psFlXvWmfBgqdP-CKxRM18kKglXBhnVm_gWmQL7dTQIWl4laqf7oPQ-ow0Hp69Es9kyDJmudOheTN_LAZ8a2csOZggnVbGAg9C4Er8xHOe2q89XxQ_szSGLOumZGQMYzDY7oamVE3jOhj7R5CPxAt9AXitCr-FFcTZeRPBFijxFfy7nthxCstLrCpsTNPn_FhaTix2IJBC4c08mC5W5BhYh01LCDZXASelii1MOh9bN28LoP9NKw6zZisclQhQmwWEwfAK00mCVkYpDeJ0YA971txzyEyEzb5EqOb9iYPymXLcP_V_-lzF1UGjtcDGiEnpyVlJk1gsds669R6AnGBOX10taWcFfIOUg-DC3xeuTDcEOtz2wXSHGuY170v086LMKD-nll9CObBX6aLLaLH9dnxqCnJu9V6atnjNHpRhg0qPgqWgLXD5ncDzUzy-bqaW6dEEhkFLNnfTFyRoEsbaRdxuwV2IN7jqha0oeh4jt80y4XJ88fCQow6zjnN_slFADyL-ViEiA4EKOIs2UosF24HpDaB7U9yZPBFb046IG8L9YO0eet63VtePHdxlu2uu8qO9qMNEjQTL21j1pHnWD0fIX6iBxW0oNrTl7wRV7wtoPU_xbw6Idlz1lrw_zW0kRFXRkdl1dFasyRp_vtpYVdPy0TvLyvdoEoPaaXWcpMcDXTmNgZYxUONuzkv3zBK651W9iqsnJxydefbbvA7DOn-PB2JWGs60wdQMaKDNmhosblCl011au7RxHGRbDjOEm-Rl2C4l0VM28pR2qV16jAaqX3hFD82VnyExFsKaeUrBhCA1jAC2QqwsNIsqWgNrPjbcAfPSjBnIOe7_qymPrnsjz7AQXXNQAwaNko_Qu5zgMmxl2fd9g07xsNZWmNmPSioTZ_Own5-DK4clk7WtNi3ESOZ8WwDYeq1PpfWXxJjVMD5SfyDRsUO47RhbIcWgcQgp9mHMqnj8MtzlkqvliCQzQizEPIyL9ukW3DDscCdpcXV9aKPuqSK2hvEthNV54OBI1czJR4xBOPAWQsviM3i5AR1ZyhTPrBqe3EPOSHwWZpu4kH2kw3IVIOy79qEcm3-b-Dc7tYXRc1DErXFuq9sPcTEX_8wRgq_m40 "Deployment Diagram for Internet Banking System - Live")
```

Comment:
The output image doesn't reflect your changes. I think you exported it before adding the default `$NODE_TYPE_MAX_CHAR_WIDTH` (tested it and it outputs OK)

Reply:
Thank you for the review, I will fix it. 
Can you please test it on [my side](https://github.com/kirchsth/C4-PlantUML/blob/extended/samples/C4CoreDiagrams.md) too?.
I'm not sure if it works 100 % correct or if it is only a problem with my browser cache (I have an non terministic behaviour that sometime the line break is not working and I cannot deflate the url itself)

Reply:
@adrianvlupu: I think my source is correct. Do you see the long tech without an linebreak like this?
![grafik](https://user-images.githubusercontent.com/21959385/103158170-46f74a00-47bb-11eb-998d-aaf1965fa3a0.png)
This is correct, the puml refernces the master branch, and this is the old implementation. As soon the branch is merged the correct 2 lines are displayed [like my merged version](https://github.com/kirchsth/C4-PlantUML/blob/extended/samples/C4CoreDiagrams.md)
![grafik](https://user-images.githubusercontent.com/21959385/103158223-cedd5400-47bb-11eb-8209-0f98c8bd66d4.png)